components/molecule/photoUploader/src/index.js

```diff
@@ -87,6 +88,10 @@ const MoleculePhotoUploader = forwardRef(
       DEFAULT_NOTIFICATION_ERROR
     )
 
+    useEffect(() => {
+      setFiles(handledFiles)
```

Comment:
Beware though, as sync props and state is usually an anti-pattern. I'd rather make it controlled/uncontrolled